app/helpers/katello/katello_urls_helper.rb

```diff
@@ -6,12 +6,13 @@ def foreman_settings_url(schema = 'http')
       ::Setting[:foreman_url].sub(%r|^.*?:|, "#{schema}:")
     end
 
-    def subscription_manager_configuration_url(host = nil, rpm = true)
-      prefix = if host&.content_source
-                 "http://#{host.content_source.hostname}"
-               else
-                 foreman_settings_url
-               end
+    def subscription_manager_configuration_url(host = nil, rpm = true, hostname = nil)
+      if hostname
+        prefix = "http://#{hostname}"
+      else
+        prefix = host&.content_source ? "http://#{host.content_source.hostname}" : foreman_settings_url
```

Comment:
can't this be `host&.content_source&.hostname or foreman_settings_url`?

Reply:
it would be missing the http:// part of the url, although now i'm wondering why its not https://    This should be https:// right?

Reply:
Should it? I think the katello-consumer rpm is not exposed via https but even if it is, for the bootstrap purpose, client at that time does not trust the certificate, the rpm contains it. Regardless of that, currently the method hardcodes that, we only allow specifying proxy hostname while keeping the same protocol. I believe this has been rebased now, is chromium alreayd resolved? Tests still seem red.

Reply:
The RPM is exposed on both HTTP and HTTPS today. But yeah, the certificate is not trusted at this step yet.

(bootstrap.py has a rather ugly hack to install RPMs from insecure HTTPS urls…)

Chromium issue should be fixed.

Reply:
Looks like we are green. @evgeni @jlsherrill can we merge?

Reply:
I have no hulk powers here.

Reply:
Wouldn't this be cleaner?
```ruby
      prefix = if hostname
                 "http://hostname"
               elsif host&.content_source
                 "http://#{host.content_source.hostname}"
               else
                 foreman_settings_url
               end
```